include/circt/Dialect/Ibis/IbisOps.td

```diff
@@ -778,6 +785,9 @@ class PortLikeOp<string mnemonic, list<Trait> traits = []> :
   code extraPortClassDefinition = ?;
   let extraClassDefinition =  extraPortClassDefinition # [{
     LogicalResult $cppClass::verify() {
+      if(getName().empty())
+        return emitOpError("port name cannot be empty");
```

Comment:
Should we default to the symbol name?

Reply:
In that case, i think the assembly format should be `ibis.port.input sym @foo` - that is, making the namehint optional. We can do that.

Reply:
I don't feel strongly about it either way.